app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,7 @@
 <resources>
     <string name="app_name">MyMediaan</string>
     <string name="discover_colleague">Discover colleague</string>
-    <string name="my_profile">My profile</string>
+    <string name="profile">My profile</string>
```

Comment:
If we use this as a title for ProfileScreen, string can be renamed to "Profile". Or perhaps to

`<string name="profile">%1$s profile</string>`

and then use it like

`stringResource(id = R.string.profile, profile.firstName)`

Reply:
I added a new one. We can't inject a string because grammatically it doesn't work for every name. 

Especially because it would not work for really important names, like 'Ivans' (it would say 'Ivanss profile') ;)